app/controllers/characters_controller.rb

```diff
@@ -378,8 +373,7 @@ def og_data
     linked << "Setting".pluralize(settings.count) + ": " + settings.join(', ') if settings.present?
     desc = [linked.join('. ')].reject(&:blank?)
     desc << generate_short(@character.description) if @character.description.present?
-    reply_posts = Reply.where(character_id: @character.id).select(:post_id).distinct.pluck(:post_id)
-    posts_count = Post.where(character_id: @character.id).or(Post.where(id: reply_posts)).uniq.count
+    posts_count = Post.where(id: Reply.where(character_id: @character.id).select(:post_id).distinct.pluck(:post_id)).count
```

Comment:
```rb
Reply.where(character_id: @character.id).select(:post_id).distinct.pluck(:post_id)
```
will by itself provide a list of distinct post IDs with the character, so we can just put a count on that (and `.distinct.count` might work also, I dunno)

Reply:
oh that's clever. yeah that makes sense